api/satellite.yaml

```diff
@@ -195,6 +195,8 @@ components:
       properties:
         party_id:
           type: string
+        party_did:
```

Comment:
Is this supplied by IShare too? If we diverge from their API to fit our needs, then we can drop the whole support from them and just stick with the EBSI TIR API